Dockerfile

```diff
@@ -9,10 +9,12 @@ RUN apk add --no-cache \
     rsync
 
 ENV VERSION 0.49
+ENV VARIATION hugo
```

Comment:
The idea would be using Docker Cloud's admin panel to build an alternative tag injecting `VARIATION=hugo_extended` (https://github.com/gohugoio/hugo)